installer/deploy/installer.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
```

Comment:
I know each service has a deploy folder. But do we need this for the installer? The same files are located in installer/manifests/installer.

Reply:
Actually those two files in installer/deploy/ should not have made it to this PR; that was a mistake. I'm removing them.

Reply:
Ok, thanks!